src/sass/layouts/_about.scss

```diff
@@ -171,13 +171,18 @@
   }
 }
 .about__btn {
-  max-width: 176px;
+  display: flex;
+  width: 176px;
```

Comment:
change to min-width. In this case button with possible longer text will look also good

Reply:
done